content/docs/user-guide/managing-external-data.md

```diff
@@ -161,6 +161,25 @@ it. So systems like Hadoop, Hive, and HBase are supported!
 
 <details>
 
+### Click for WebHDFS
+
+```dvc
```

Comment:
Just missing adding WebHDFS to the bullet list earlier in this doc., maybe in the same line as HDFS.

Reply:
Also missing similar updates to the ext deps page.